crates/cli/Cargo.toml

```
@@ -21,6 +21,6 @@ name = "soldeer"
 path = "src/main.rs"
 
 [dependencies]
-soldeer-commands = { path = "../commands", version = "0.4.1" }
+soldeer-commands = { path = "../commands", version = "0.5.0" }
```

```suggestion
soldeer-commands = { path = "../commands", version = "0.5" }
```

This is maybe easier so that we don't have to edit every time we bump the patch number.